### PR TITLE
[fix] Optimize speech imports

### PR DESCRIPTION
**Pull request by Thoughtseize1**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed [Sample Guidelines from AUTHORING_GUIDE.MD](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md)
- [ ] README is updated to include [all relevant information](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#readme-file)
- [x] **Tests** pass:   `nox -s py-3.9` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [x] **Lint** pass:   `nox -s lint` (see [Test Environment Setup](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#test-environment-setup))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Review by m-strzelczyk — APPROVED** [review]
Let's have @msampathkumar also review this change. Before you merge.

File: speech/snippets/speech_adaptation_beta.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google LLC
+# Copyright 2024 Google LLC
```
Comment (msampathkumar) [inline speech/snippets/speech_adaptation_beta.py:1]: Please roll back copyright year change

File: speech/snippets/speech_adaptation_beta.py [hunk 2]
```diff
@@ -80,20 +80,5 @@ def sample_recognize(storage_uri: str, phrase: str) -> speech.RecognizeResponse:
...
-    )
-    parser.add_argument("--phrase", type=str, default="Brooklyn Bridge")
-    args = parser.parse_args()
-
-    sample_recognize(args.storage_uri, args.phrase)
-
-
 if __name__ == "__main__":
-    main()
+    recognition_response = sample_recognize()
```
Comment (msampathkumar) [inline speech/snippets/speech_adaptation_beta.py:84]: Why store response in `recognition_response` ?

File: speech/snippets/speech_model_adaptation_beta.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2024 Google LLC
```
Comment (msampathkumar) [inline speech/snippets/speech_model_adaptation_beta.py:1]: Please roll back this! Copyright is fixed, on the date the file is created.

Comment (msampathkumar) [reply speech/snippets/speech_model_adaptation_beta.py:1]: Lets rollback the changes for this file! Existing scripts looks good to me at https://cloud.google.com/speech-to-text/docs/adaptation

**Review by msampathkumar — CHANGES_REQUESTED** [review]
Mostly, LGTM. Added minor comments. PTAL.

File: speech/snippets/speech_adaptation_beta.py (hunk 2 above)
Comment (Thoughtseize1) [reply speech/snippets/speech_adaptation_beta.py:84]: I agree. It is not necessary.

File: speech/snippets/speech_adaptation_beta.py (hunk 1 above)
Comment (Thoughtseize1) [reply speech/snippets/speech_adaptation_beta.py:1]: Done✅

**Review by holtskinner — APPROVED** [review]

**Review by msampathkumar — APPROVED** [review]
LGTM.

**Pull request merged**